libraries/AP_HAL_F4Light/boards/f4light_MatekF405_CTR/1_read_me.md

```diff
@@ -31,9 +31,9 @@ Firmware supports connection to built-in OSD with CT from my MinimOSD (https://g
 
 How to get voltage/current reading(tested on omnibus, should work on other targets to):
 - BAT_MONITOR 4
-- BAT_VOLT_PIN 10
-- BAT_CURR_PIN 11
-- BAT_VOLT_MULT 11.0
+- BAT_VOLT_PIN 11
+- BAT_CURR_PIN 10
+- BAT_VOLT_MULT 10.95
```

Comment:
difference less 5%, so most likely the deviation of a particular specimen.

Reply:
I aggree issue is not in deviation from 11.0 to 10.95 for BAT_VOLT_MULT  but 
wiki : BAT_VOLT_PIN 10 -> tested ok :BAT_VOLT_PIN 11
and
wiki : BAT_CURR_PIN 11 -> tested ok : BAT_CURR_PIN 10

(why the github diff no showing this ... doesn't matter)

Reply:
this part of Readme can be safely deleted because this parameters are set by default to correct values now.

Reply:
nice ! thanks.  So I close pull request.  thanks